### PR TITLE
FRW-998 updated PHP version

### PR DESCRIPTION
**Pull request by a-sabaa**
#### Overview

- Ticket: https://spryker.atlassian.net/browse/FRW-998

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -335,12 +335,12 @@ jobs:
                   docker/sdk testing codecept run -c codeception.functional.yml
 
 
-    php-80-mariadb-glue-alpine:
-        name: "PHP 8.0 / MariaDB / Glue / Alpine"
+    php-82-mariadb-glue-alpine:
```
Comment (asmarovydlo) [inline .github/workflows/ci.yml:338]: Everything that was 8.0 should be 8.1, everything that was 8.1 should be 8.2 please check all other places

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -478,12 +478,12 @@ jobs:
...
 
-    php-80-postgres-functional-debian:
-        name: "PHP 8.0 / PostgreSQL / Functional / Debian"
+    php-82-postgres-functional-debian:
+        name: "PHP 8.2 / PostgreSQL / Functional / Debian"
         runs-on: ubuntu-22.04
         env:
             PROGRESS_TYPE: plain
-            SPRYKER_PLATFORM_IMAGE: spryker/php:8.0-debian
+            SPRYKER_PLATFORM_IMAGE: spryker/php:8.2
```
Comment (asmarovydlo) [inline .github/workflows/ci.yml:486]: Debian should stay (It is debian validation job). (+ version should be 8.1)

File: README.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # Spryker B2B Marketplace Demo Shop
 [![Build Status](https://github.com/spryker-shop/b2b-demo-marketplace/actions/workflows/ci.yml/badge.svg?branch=master)](https://github.com/spryker-shop/b2b-demo-marketplace/actions?query=branch:master)
 [![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/spryker-shop/b2b-demo-marketplace/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/spryker-shop/b2b-demo-marketplace/?branch=master)
-[![Minimum PHP Version](https://img.shields.io/badge/php-%3E%3D%208.0-8892BF.svg)](https://php.net/)
+[![Minimum PHP Version](https://img.shields.io/badge/php-%3E%3D%208.-8892BF.svg)](https://php.net/)
```
Comment (asmarovydlo) [inline README.md:4]:
```suggestion
[![Minimum PHP Version](https://img.shields.io/badge/php-%3E%3D%208.1-8892BF.svg)](https://php.net/)
```

File: deploy.ci.acceptance.mariadb.prefer-lowest.dynamic-store.yml [hunk 4]
```diff
@@ -6,7 +6,7 @@ tag: '1.0'
 environment: docker.ci
 pipeline: docker.ci.acceptance.dynamic-store
 image:
-    tag: spryker/php:8.1
+    tag: spryker/php:8.2
```
Comment (asmarovydlo) [inline deploy.ci.acceptance.mariadb.prefer-lowest.dynamic-store.yml:9]: Prefer lowest deploy files should stay 8.1

File: phpcs.xml [hunk 5]
```diff
@@ -7,7 +7,7 @@
         All sniffs in ./Sniffs will be auto loaded
     </description>
 
-    <config name="php_version" value="80000"/>
+    <config name="php_version" value="80100"/>
```
Comment (asmarovydlo) [inline phpcs.xml:10]: Let's be consistent with other demoshops
```suggestion
    <config name="php_version" value="80122"/>
```

**Review by asmarovydlo — CHANGES_REQUESTED** [review]

File: deploy.ci.functional.mariadb.prefer-lowest.dynamic-store.yml [hunk 6]
```diff
@@ -6,7 +6,7 @@ tag: '1.0'
 environment: docker.ci
 pipeline: docker.ci.functional.dynamic-store
 image:
-    tag: spryker/php:8.1
+    tag: spryker/php:8.2
```
Comment (asmarovydlo) [inline deploy.ci.functional.mariadb.prefer-lowest.dynamic-store.yml:9]: Prefer-lowest jobs should stay 8.1
```suggestion
    tag: spryker/php:8.1
```

**Review by asmarovydlo — APPROVED** [review]

**Review by FRITS-Kh — APPROVED** [review]
FE part

**Pull request merged**